package.json

```diff
@@ -31,6 +31,9 @@
   },
   "jest": {
     "verbose": true,
+    "setupFilesAfterEnv": [
+      "./setupTests.js"
```

Comment:
I needed to wire all this up to write my specs else I was getting regenerator-runtime/runtime related errors